src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/EnvironmentSetupHelper.cs

```diff
@@ -194,7 +200,102 @@ public void SetupEnvironment(AzureModule mode)
 #endif
         }
 
-        private void SetupAzureEnvironmentFromEnvironmentVariables(AzureModule mode)
+        public void SetEnvironmentVariableFromCredentialFile()
+        {
+            var filePath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile), ".azure/testcredentials.json");
```

Comment:
In Path.Combine usage, you pass in the root path as the first parameter, and each subsequent path as an additional parameter.  So for example,

```c#
Path.Combine(@"C:\foo", "bar", "baz")
```

results in ```C:\foo\bar\baz"

Reply:
done